README.md

```diff
@@ -6,22 +6,52 @@
 
 [![codecov.io](http://codecov.io/github/ahnlabb/BioformatsLoader.jl/coverage.svg?branch=master)](http://codecov.io/github/ahnlabb/BioformatsLoader.jl?branch=master)
 
-## Usage
+## Dependencies
 
-Requires `bioformats_package.jar`
+Depends on `bioformats_package.jar`
+
+## Install
+
+```julia
+Pkg.add("https://github.com/ahnlabb/BioformatsLoader.jl") # Change if forked
+Pkg.build("BuildformatsLoader") # Downloads bioformats_package.jar and ome.xsd
+```
+
+## Setup Environment
+
+Set the environmental variable `JULIA_COPY_STACKS` to `1`. On Linux and Mac, this can be done by invoking julia in the following way:
+
+```bash
+$ JULIA_COPY_STACKS=1 julia
+```
+
+## Basic Usage
 
 ```julia
-Pkg.clone("https://github.com/ahnlabb/BioformatsLoader.jl")
+using BioformatsLoader
+BioformatsLoader.init() # Initializes JavaCall with opt and classpath
+image = bf_import("file.msr") # Import the image file.msr
+collect(image[1].data) # Get a standard Julia 2D array
```

Comment:
Do you find that you often need to make a copy of the data instead of using the `ReshapedArray{T,2,ReinterpretArray{T,1}}` directly? If this is the case maybe I should consider changing the API but these views should be transparent and I don't really want to encourage making copies unnecessarily.

Reply:
For some applications, this may be necessary. For example, I was using this across PyCall to load data into Napari:
https://www.youtube.com/watch?v=xyNrnGh1ugI